travis/setup_conda_windows.sh

```diff
@@ -6,12 +6,14 @@ if [[ $DEBUG == True ]]; then
 fi
 
 if [[ -z "${MINICONDA_VERSION}" ]]; then
-    MINICONDA_VERSION=4.5.4
+    MINICONDA_VERSION=4.6.14
 fi
 
 echo "installing miniconda3"
 choco install miniconda3 --params="'/AddToPath:1'" --version="$MINICONDA_VERSION";
-export PATH="/c/tools/miniconda3/scripts:/c/tools/miniconda3/:$PATH";
+export PATH="/c/tools/miniconda3/scripts:/c/tools/miniconda3/bin:$PATH"
+conda init bash
```

Comment:
Adding the following and removing `/c/tools/miniconda3/bin` from the above PATH got me a lot further.

```suggestion
conda init bash
source /c/Users/travis/.bash_profile
```

Reply:
After the above I then get:

```
++++++ export PATH
++++++ /c/tools/miniconda3/Scripts/conda.exe install conda
Collecting package metadata: ...working... done
Solving environment: ...working... failed
SpecsConfigurationConflictError: Requested specs conflict with configured specs.
  requested specs: 
  pinned specs: 
    - conda==4.7
Use '\''conda config --show-sources'\'' to look for '\''pinned_specs'\'' and '\''track_features'\''
configuration parameters.  Pinned specs may also be defined in the file
C:\tools\miniconda3\conda-meta\pinned.
```

Reply:
Found some old code I had in my project's travis that were manually removing the `defaults` channel. This was before strict channel priority was a thing. I'm removing it to see if that changes anything.

Reply:
I'm still getting the above error. I started talking to some folks on the conda-forge gitter for help. They are wondering if this is a bug. @mwcraig see https://gitter.im/conda-forge/conda-forge.github.io